litestar/contrib/sqlalchemy/alembic/commands.py

```diff
@@ -71,10 +73,24 @@ class AlembicSpannerImpl(DefaultImpl):
     __dialect__ = "spanner+spanner"
 
 
+def _get_database_migration_plugin(app: Litestar) -> SQLAlchemyPlugin | SQLAlchemyInitPlugin:
+    """Retrieve a database migration plugin from the Litestar application's plugins.
+
+    This function attempts to find and return either the SQLAlchemyPlugin or SQLAlchemyInitPlugin.
+    If neither plugin is found, it raises an ImproperlyConfiguredException.
+    """
+    for type_ in SQLAlchemyPlugin, SQLAlchemyInitPlugin:
+        with suppress(KeyError):
```

Comment:
```suggestion
        if type_ in app.plugins:
```

Reply:
> Suggested change
>         with suppress(KeyError):
>         if type_ in app.plugins:

I think this does not exactly do what I intend to? It always seems to trigger `ImproperlyConfiguredException`